frontend/src/lib/components/Cards/InsightCard/InsightDetails.tsx

```diff
@@ -158,8 +158,8 @@ function SeriesDisplay({
     const hasBreakdown =
         isInsightQueryWithBreakdown(query) &&
         query.breakdownFilter != null &&
-        query.breakdownFilter.breakdown_type != null &&
-        query.breakdownFilter.breakdown != null
+        ((query.breakdownFilter.breakdown_type != null && query.breakdownFilter.breakdown != null) ||
+            Array.isArray(query.breakdownFilter.breakdowns))
```

Comment:
Is it intentional that we check for any array (also empty) here and below we look for arrays with values?

Reply:
It is incorrect. Changed to a length condition.